src/pages/workspace/accounting/netsuite/export/NetSuiteExportConfigurationPage.tsx

```diff
@@ -79,7 +82,9 @@ function NetSuiteExportConfigurationPage({policy}: WithPolicyConnectionsProps) {
             description: translate('common.date'),
             onPress: () => Navigation.navigate(ROUTES.POLICY_ACCOUNTING_NETSUITE_DATE_SELECT.getRoute(policyID)),
             brickRoadIndicator: config?.errorFields?.exportDate ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined,
-            title: config?.exportDate ? translate(`workspace.netsuite.exportDate.values.${config.exportDate}.label`) : undefined,
+            title: config?.exportDate
+                ? translate(`workspace.netsuite.exportDate.values.${config.exportDate}.label`)
+                : translate(`workspace.netsuite.exportDate.values.${CONST.NETSUITE_EXPORT_DATE.LAST_EXPENSE}.label`),
```

Comment:
Hmmm... I don't think we should be using the option text as a "title" here for the default case. We should just always have it selected by default and keep `undefined` as the secondary option.

Reply:
Also, I changed this already to use the generic `Export date` label, so this will run into conflicts. I think we should use the generic one so as to keep our labels as DRY as possible.